applications/rag/variables.tf

```diff
@@ -398,7 +398,7 @@ variable "gpu_pools" {
     name               = "gpu-pool-l4"
     machine_type       = "g2-standard-24"
     autoscaling        = true
-    min_count          = 1
+    min_count          = 0
```

Comment:
should we have 1 min worker pre-created so the ray job isn't super slow?

Reply:
From my testings, it took around 5 mins to scale up one worker node from 0. I feel it's not very long and sounds like a reasonable trade for cost. But I don't have strong opinion.

Reply:
We could also set the [initial node count](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_node_pool#initial_node_count) to 1 and enable autoscaling from 0 -> 3 which would allow it to scale down to 0 when nothing was running but pre-create the node at the beginning when we know we are about to use it. 

That would be a bit more of an invasive change since we'd have to plumb a new variable into the template.

Reply:
Also, this change only applies when creating a standard cluster (which is no longer the default) since you don't create node pools on autopilot.

Reply:
Looking again, it looks like the initial_node_count defaults to 1 so that should already work nicely.

Reply:
Yes, for standard cluster, it creates a gpu pool with 1 node default since initial_node_count is set.